builtin/providers/aws/resource_aws_autoscaling_attachment.go

```diff
@@ -0,0 +1,110 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/autoscaling"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsAutoscalingAttachment() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsAutoscalingAttachmentCreate,
+		Read:   resourceAwsAutoscalingAttachmentRead,
+		Delete: resourceAwsAutoscalingAttachmentDelete,
+
+		Schema: map[string]*schema.Schema{
+			"group_name": &schema.Schema{
```

Comment:
I would call the attribute `autoscaling_group_name` to be consistent with other resources in the same group.